bower.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-copy-to-clipboard",
+  "version": "1.0.1",
+  "homepage": "https://github.com/nkbt/react-copy-to-clipboard",
+  "authors": [
+    "Nik Butenko <nik@butenko.me> (http://butenko.me/)"
+  ],
+  "description": "Copy-to-clipboard React component",
+  "main": "lib/index.js",
```

Comment:
Thanks for the context, yeah, I'm pretty new to webpack and the ecosystem of JS bundlers.  The only reason I care about Bower is that it allows React components to be used in the Rails asset pipeline through https://rails-assets.org/

Eventually my team will probably move to an entirely separated system of Rails APIs on the backend and Webpack/React/Flux on the frontend and then this will stop being a problem, but until then, I'd like to use the systems that are already in place as much as I can.